controller/rest/repository.go

```diff
@@ -21,11 +21,13 @@ import (
 )
 
 const (
-	repositoryDefaultTag      = "latest"
-	maxRepoScanTasks          = 32
-	repoScanTimeout           = time.Minute * 20
-	repoScanLingeringDuration = time.Second * 30
-	repoScanLongPollTimeout   = time.Second * 30
+	repositoryDefaultTag = "latest"
+	// maxConcurrentRepoScanTasks determines the maximum number of scan tasks that can run concurrently.
+	// Limiting it to 4 is a safer choice to prevent excessive memory usage.
+	maxConcurrentRepoScanTasks = 4
+	repoScanTimeout            = time.Minute * 20
+	repoScanLingeringDuration  = time.Second * 30
+	repoScanLongPollTimeout    = time.Second * 30
```

Comment:
Make these configurable?

Reply:
Do you mean config in the controller entry point?